.env.example

```diff
@@ -9,4 +9,4 @@ APP_PATH=<LOCAL_PATH/App.app>
 ALLURE_RESULTS_DIR=tests/.reports/allure-results
 
 ## Choose the environment in the configs folder
-ENVIRONMENT=dev-comm #dev-ent
\ No newline at end of file
+ENVIRONMENT=dev-direct
```

Comment:
Lets set this to `dev-comm` - `this.getCreateAidOptions());` is commented out still in `identifierService.ts` so we can pretend to use the backer and just show it in the UI.

I will fix the routing for the backer to allow our local KERIA to access it next week hopefully, and we can uncomment that.

Afterwards we will have discussions about how we can support witness and backers at the same time in the wallet via the UI.